src/pages/home/containers/Landing.tsx

```diff
@@ -33,10 +36,10 @@ export function Landing() {
           </Heading>
           <Text fontSize="1.6em">CourseUp makes it simple to browse and schedule UVic Courses</Text>
           <Flex alignItems="center" mt="1rem" as="a" w="fit-content" target="_blank" href="https://www.vikelabs.ca">
-            <Text color="#4C6EA5" fontSize={{ base: '1em', md: '1.5em' }}>
+            <Text fontSize={{ base: '1em', md: '1.5em' }} color={isDarkMode ? '#799EC1' : '#4C6EA5'}>
               <span>{hands[handIndex]}</span> Built by students @
             </Text>
-            <Text fontWeight="bolder" fontSize="1.75em" color="#222B49" ml="1">
+            <Text fontWeight="bolder" fontSize="1.75em" color={isDarkMode ? '#B0C3DA' : '#222B49'} ml="1">
```

Comment:
A thought I had is it might be good to define these colours in the theme instead of throwing hex values around :thinking: